packages/cli/src/actions/getShareBalances.ts

```diff
@@ -0,0 +1,83 @@
+import SDK, { util } from "@zeitgeistpm/sdk";
+
+type Options = {
+  addressOrSeed: string;
```

Comment:
We probably don't need to accept seed in this function, don't really see why someone would want to input a seed here instead of the address.

Reply:
only for consistency, so as not to require a different input format when reusing previously entered commands.